### ISSUE TITLE
Tarea 5 pregunta 1 subpregunta 4.2

### ISSUE THREAD

**Issue by MaxAl100** [IIC2613-Inteligencia-Artificial-2024-2/Syllabus#103]
En la subpregunta 4.2 de la pregunta 1 se pide
> Define nuevamente los mismos MLP utilizados en la pregunta 1.1.1. y entrenalos usando los embeddings generados por ResNet50

Pero en la pregunta 1.1.1 no se crean los MLP, se crean para la pregunta 1.2. ¿Puedo asumir que se refieren a estos?

**Comment by jinfantel** [comment]
Sí, efectivamente hace referencia a la implementación de una MLP de la sección 1.2. Asume que se refiere a esos.
